mysql-test/suite/tianmu/t/multi_table_delete.test

```diff
@@ -0,0 +1,314 @@
+  `c_time` time DEFAULT NULL COMMENT 'time',
+  `c_char` char(10) DEFAULT NULL COMMENT 'char',
+  `c_varchar` varchar(10) DEFAULT NULL COMMENT 'varchar',
+  `c_blob` blob COMMENT 'blob',
+  `c_text` text COMMENT 'text',
+  `c_longblob` longblob COMMENT 'longblob'
+) ENGINE=TIANMU;
+
+CREATE TABLE `t2` (
+ `c_tinyint` tinyint(4) DEFAULT NULL COMMENT 'tinyint',
+  `c_smallint` smallint(6) DEFAULT NULL COMMENT 'smallint',
+  `c_mediumint` mediumint(9) DEFAULT NULL COMMENT 'mediumint',
+  `c_int` int(11) DEFAULT NULL COMMENT 'int',
+  `c_bigint` bigint(20) DEFAULT NULL COMMENT 'bigint',
+  `c_float` float DEFAULT NULL COMMENT 'float',
+  `c_double` double DEFAULT NULL COMMENT 'double',
+  `c_decimal` decimal(10,5) DEFAULT NULL COMMENT 'decimal',
+  `c_date` date DEFAULT NULL COMMENT 'date',
+  `c_datetime` datetime DEFAULT NULL COMMENT 'datetime',
+  `c_timestamp` timestamp NULL DEFAULT NULL COMMENT 'timestamp',
+  `c_time` time DEFAULT NULL COMMENT 'time',
+  `c_char` char(10) DEFAULT NULL COMMENT 'char',
+  `c_varchar` varchar(10) DEFAULT NULL COMMENT 'varchar',
+  `c_blob` blob COMMENT 'blob',
+  `c_text` text COMMENT 'text',
+  `c_longblob` longblob COMMENT 'longblob'
+) ENGINE=TIANMU;
+
+--echo #
+--echo # DELETE tb1_name,tb2_name FROM table_references for Large amount of data
+--echo #
+
+load data infile '../../std_data/tianmu/bigdata4load' into table t1 fields terminated by '|';
+select count(*) from t1;
+select max(c_int) from t1;
+select min(c_int) from t1;
+select avg(c_int) from t1;
+select sum(c_int) from t1;
+load data infile '../../std_data/tianmu/bigdata4load' into table t2 fields terminated by '|';
+select count(*) from t2;
+select max(c_int) from t2;
+select min(c_int) from t2;
+select avg(c_int) from t2;
+select sum(c_int) from t2;
+
+delete t1,t2 from t1,t2;
+
+select count(*) from t1;
+select max(c_int) from t1;
+select min(c_int) from t1;
+select avg(c_int) from t1;
+select sum(c_int) from t1;
+
+select count(*) from t2;
+select max(c_int) from t2;
+select min(c_int) from t2;
+select avg(c_int) from t2;
+select sum(c_int) from t2;
+
+--echo #
+--echo # DELETE tb1_name,tb2_name FROM table_references for select/insert/Aggregate query
+--echo #
+
+insert into t1(c_tinyint,c_smallint,c_mediumint,c_int,c_bigint,c_float,c_double,c_decimal,c_date,c_datetime,c_timestamp,c_time,c_char,c_varchar,c_text)
+values(105,105,105,105,105,5.2,10.88,105.083,'2016-02-25','2016-02-25 10:20:01','2016-02-25 05:20:01','10:20:01','stoneatom','hello','bcdefghijklmn');
+select * from t1;
+select count(*) from t1;
+select max(c_int) from t1;
+select min(c_int) from t1;
+select avg(c_int) from t1;
+select sum(c_int) from t1;
+
+insert into t2(c_tinyint,c_smallint,c_mediumint,c_int,c_bigint,c_float,c_double,c_decimal,c_date,c_datetime,c_timestamp,c_time,c_char,c_varchar,c_text)
+values(105,105,105,105,105,5.2,10.88,105.083,'2016-02-25','2016-02-25 10:20:01','2016-02-25 05:20:01','10:20:01','stoneatom','hello','bcdefghijklmn');
```

Comment:
Adds some test cases for unicode. such as  insert unicode chars, in values, table name, or etc.